docs/components/modeler/bpmn/multi-instance/multi-instance.md

```diff
@@ -94,6 +94,14 @@ target: output
 
 A `completionCondition` defines whether the multi-instance body can be completed immediately when the condition is satisfied. It is a [boolean expression](/components/concepts/expressions.md#boolean-expressions) that will be evaluated each time the instance of the multi-instance body completes. Any instances that are still active are terminated and the multi-instance body is completed when the expression evaluates to `true`.
 
+The BPMN 2.0 specification defines the following properties of a multi-instance body instance:
```

Comment:
🔧  `multi-instance body instance` is correct, but might be a bit confusing to newcomers. Let's just call it `multi-instance body` like in the paragraph above.